src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +37,24 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={filterTitle}
+        onChange={(event) => onFilterTitleChange(event.target.value)}
       />
       <span className="icon is-left">
         <i className="fas fa-magnifying-glass" />
       </span>
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
-        />
-      </span>
+      {filterTitle !== '' && (
+        <span className="icon is-right" style={{ pointerEvents: 'all' }}>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
you may solve this lint issue just add arial-lable attribute `aria-label="clear search"`

The rule is triggered by the role="button" attribute.  it needs a text label for accessibility reasons (so screen readers know what to read out, for example). To comply with the rule, you can either give the "button" textual content or add an `aria-label attribute`

Reply:
It was here from beginning, so I decided not to fix it, but thanks for advice)